lyte/bg-image-layzload.js

```diff
@@ -0,0 +1,52 @@
+/*
+*  Original source taken from: https://imagekit.io/blog/lazy-loading-images-complete-guide/#lazy-loading-css-background-images
+*/
+
+document.addEventListener("DOMContentLoaded", function() {
+  var wyl_lazyloadImages;
+
+  if ("IntersectionObserver" in window) {
+    wyl_lazyloadImages = document.querySelectorAll(".wyl-lazy");
+    var imageObserver = new IntersectionObserver(function(entries, observer) {
+      entries.forEach(function(entry) {
+        if (entry.isIntersecting) {
+          var image = entry.target;
+          image.classList.remove("wyl-lazy");
```

Comment:
Might not understand, but it seems you're not setting img.src like you're doing below when intersectionobserver is not present softmanro ? Or is the image loaded in another manner?

Reply:
Actually the code when the IntersectionObserver is correct, while the bit when it is not available is somewhat inadequate. Setting the img.src is really not needed, as lyte.js already handles setting the background-image src to data-src attribute; here we simply need to remove the “wyl-lazy” css class which prevented loading the bg image.